consumer/nnrf.go

```diff
@@ -348,3 +380,52 @@ func SendDeregisterNFInstance() (*models.ProblemDetails, error) {
 		return nil, openapi.ReportError("server no response")
 	}
 }
+
+func SendCreateSubscription(nrfUri string, nrfSubscriptionData models.NrfSubscriptionData) (nrfSubData models.NrfSubscriptionData, problemDetails *models.ProblemDetails, err error) {
+	logger.ConsumerLog.Debugf("Send Create Subscription")
```

Comment:
Adding what NF is being subscribed with some details shall be helpful in the log

Reply:
Added NF type.